### DISCUSSION TITLE
Thread <unnamed> panicked

### DISCUSSION THREAD

**Discussion by fabianwohlfart** [nuxt/nuxt#24769] [Questions]
Hi guys,

Yesterday I ran into a problem on our production server.
When building my application I get this Rust error.

The guys from server administration can not explain it, but say it has to do with Rollup (which is used by Vite) and I am not dev ops enough to understand this.
Maybe one of you has an idea what is going on and how we can fix it? – Or what we can try to work around it?

Error in all versions of Nuxt (3.6.5 onwards, Vite 4.3.9 onwards)

```
thread '<unnamed>' panicked at /usr/local/cargo/registry/src/index.crates.io-6f17d22bba15001f/stacker-0.1.15/src/lib.rs:416:13:
assertion `left == right` failed
  left: 2
 right: 0
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
fatal runtime error: failed to initiate panic, error 5
```

**Comment by Nikoxx99** [comment]
Same here
